File: fizzbuzz/fizzbuzz.py
def fizzbuzz(n):
    response = ''
    if n % 3 == 0:
        response += 'fizz'
    if n % 5 == 0:
        response += 'buzz'

    if not response:
        response = n
    return response

for n in range(100):
    print fizzbuzz(n)

File: fizzbuzz/go/fizzbuzz.go
package main

import "fmt"

func main() {
	for n := 1; n <= 100; n++ {
		s := ""
		if n % 3 == 0 {
			s = s + "fizz"
		}
		if n % 5 == 0 {
			s += "buzz"
		}
		if s == "" {
			fmt.Printf("%d\n", n)
		} else {
			fmt.Printf("%s\n", s)
		}
	}
}
